WordPress/Classes/Models/ReaderPost.m

```diff
@@ -505,4 +506,17 @@ - (NSDictionary *)railcarDictionary
     return nil;
 }
 
+- (void) didSave {
+    [super didSave];
+
+    // When saving a post, changes are not reflected in Reader Discover because of the usage of
+    // ReaderCard as the Core Data object. By "faking" this change we make sure Reader Discover
+    // reflects a change on a post.
+    // We can confidently set `topics` to NULL given a card represents a post OR a list of topics.
```

Comment:
If I understand correctly: because the card is only showing posts, its okay to make a change to topics since topics should be empty anyway.  Does that sound right?  This jives with my understanding of how ReaderCards are created and how they work but want to confirm.

Reply:
Yes!

A card can be only a post OR a topic. Do you have any tips on how to make the comment clear?

Reply:
It wasn't really unclear, I just was being extra sure I understood. :) I haven't looked at the card code much.  But since you ask, if I were going to nitpick, maybe something like:  
```
A ReaderCard can have either a post, or a list of topics, but not both. Since this card has a post, we can confidently set `topics` to NULL.
```
But I think what's there is probably fine as is. :)